libraries/chain/include/eosio/chain/platform_timer.hpp

```diff
@@ -26,17 +26,21 @@ struct platform_timer {
       callback still execute if the timer expires and stop() is called nearly simultaneously.
       However, set_expiration_callback() is synchronized with the callback.
    */
-   void set_expiration_callback(void(*func)(void*), void* user) {
+   void set_expiration_callback(void(*func)(void*), void* user, bool appending = false) {
```

Comment:
If the approach is deemed OK, I will replace this default parameter with regular parameter, and define an enum for the boolean, like `{muilt_callbacks_allowed, muilt_callbacks_disallowed}`.